Adaptors/MongoDB/src/ResultTable.cs

```diff
@@ -94,6 +94,30 @@ public class ResultTable : IResultTable
     }
   }
 
+  async Task<Core.Common.Storage.Result> IResultTable.GetResult(string            sessionId,
+                                                                string            key,
+                                                                CancellationToken cancellationToken)
```

Comment:
=default

Reply:
According to R#, not necessary here !

Reply:
I understand why R# says that it's not necessary. But I'm not sure it's true.
[https://stackoverflow.com/a/8909925](https://stackoverflow.com/a/8909925)

Reply:
Optional parameters are given their default value by the caller, not the callee. And the caller does not know what is the actual type of the object, and thus, can only take the default value specified in the base declaration.